docs/setup_bothan.md

```diff
@@ -0,0 +1,52 @@
+# Bothan Installation Guide
+
+## Prerequisites
+
+Before building and running Bothan, ensure the following dependencies are installed:
+
+- [Clang](https://clang.llvm.org/get_started.html)
+- [Rust](https://www.rust-lang.org/tools/install)
+
+## Building Bothan
+
+To install the Bothan CLI, follow the steps below:
+
+- Clone or navigate to the bothan-api repository.
+- Run the following command to build and install the Bothan CLI
+  binary:
+
+```bash 
+cargo install --path bothan-api/server-cli --bin bothan
+```
+
+This will compile and install the bothan executable
+
+## Configuration Setup
+
+Once Bothan is installed, the next step is to initialize the configuration file. By default, the configuration file will
+be created in the `$HOME/.bothan/ directory`.
+
+- Initialize the configuration file using the command:
+
+```bash 
+bothan config init
+```
+
+- Open the configuration file in any text editor of your choice and adjust the parameters as necessary
```

Comment:
should we state that some sources need api_key for bothan to be able to start

Reply:
it won't be necessary since the service can be run without the api key